index.html

```diff
@@ -0,0 +1,146 @@
+         <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="top-navbar">
+      <div class="logo">
+        <i class="fa fa-university fa-4x" aria-hidden="true">
+        <a href="./index.html"></a></i>
+        <h2>Stanmega-tech <span>Web Development</span></h2>
+      </div>
+
+      <ul class="nav-icons">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+      </ul> 
+      <div id="campaign">
+      <i class="fa fa-university" aria-hidden="true" >
+      <a href="#">SMT Campaign</a></i>
+    </div>
+  </div>
+  </nav>
+  <!--Header design-->
+  <header>
+    <span class="menu">
+      <i class="fas fa-bars fa-2x" id="menu"></i></span>
+    <div class="headline">
+    <span id="welcome">"Hello! Welcome to Stanmega-Tech"</span>
+    <h1 class="h1text">
+      WEB DEVELOPMENT
+      <span class="h1-span">COURSE</span>
+      <span class="h1-span">2022</span>
+    </h1>
+    <p>
+      Do you want to become a web developer? If yes, then this course is
+      just for you. The program's duration is 10 months of intensive learning with practicals. We start a new class every 2 months, so if you don't
+      want to miss the start date for the next class, then sign up and join
+      us now.
+    </p>
+    <h2 id="next-class"
+      >Our next class starts:
+      <span id="start-date">2022.05.23.(MON)</span></h2>
+    <span id="location"><a href="#"> Stanmega-Tech online Link</a></span>
+    <button type="button" class="access">Get Access Now</button>
+  </div>
+</header>
+
+<!--Our Curriculum-->
+<main>
+<section id="program">
+  <h2 class="text">Our Programs | Curriculum</h2>
+  <div class="text-line"></div>
+  <div id="languages-container">
+    <div class="technologies">
+       <i class="fa brands fa-html5 fa-3x"></i>
+       <h3>HTML</h3>
+          <p>Start with HTML, Hypertext Markup Language.</p>
+      </div>
+    <div class="technologies">
+        <i class="fab fa-css3-alt fa-3x"></i>
+        <h3>CSS</h3>
+          <p>Style a website with CSS, Cascading Style Sheets.</p>
+      </div>
+    <div class="technologies">
+        <i class="fab fa-js fa-3x"></i>
+        <h3>JavaScript</h3>
+          <p>Make your websites interactive with JavaScript.</p>
+    </div>
+    <div class="technologies">
+        <i class="fab fa-git-alt fa-3x"></i>
+        <h3>Git/Github</h3>
+          <p>Learn how to use Git, a version control system.</p>
+            </div>
+    <div class="technologies">
+        <i class="fa fa-terminal fa-2x" aria-hidden="true"></i>
+        <h3>CLI</h3>
+          <p>Learn how to use the command line interface.</p>
+        </div>
+      </div>
+        <button type="button">Join our course 2022</button>
+        <span id="see-program">SEE THE WHOLE PROGRAM</span>
+</section>
+<section id="featured">
```

Comment:
As the previous reviewer mentioned, the section is not properly aligned in the center. Kindly make the necessary changes so that your project meets the requirements 🤓

Reply:
Addressed 🤓 Well done!